solutions/arrays/shuffle-array.js

```diff
@@ -0,0 +1,36 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+
+function shuffle(arr){
+    //Implement this function
+	let leftIndex=0,rightIndex=arr.length/2
+	let answer=[]

```

Comment:
@DevratAgarwal  variable names can be more intuitive. Instead of `answer`, lets name it as `shuffledArray`.